src/factories/series/Column.ts

```diff
@@ -0,0 +1,92 @@
+module n3Charts.Factory.Series {
```

Comment:
Overall I'm not particularly found of using abbrevations... It usually leads to obfuscation super quickly ^^